tests/src/test/java/org/eclipse/hono/tests/registry/AbstractDeviceRegistryAmqpSupportIT.java

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) 2018 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 1.0 which is available at
+ * https://www.eclipse.org/legal/epl-v10.html
+ *
+ * SPDX-License-Identifier: EPL-1.0
+ */
+
+package org.eclipse.hono.tests.registry;
+
+import io.vertx.core.Future;
+import io.vertx.core.Vertx;
+import io.vertx.ext.unit.TestContext;
+import org.eclipse.hono.client.HonoClient;
+import org.eclipse.hono.client.impl.HonoClientImpl;
+import org.eclipse.hono.config.ClientConfigProperties;
+import org.eclipse.hono.tests.IntegrationTestSupport;
+
+/**
+ * Abstract class to support integration tests for the device registry.
+ *
+ */
+public abstract class AbstractDeviceRegistryAmqpSupportIT {
```

Comment:
there doesn't seem to be any need to extend this helper class, is there?
So I guess the test classes shouldn't. Instead this class should be renamed to `DeviceRegistryAmqpTestSupport` and get a private default constructor.

Reply:
Good proposal, will change it.